src/test/java/utils/ApiUtils.java

```diff
@@ -0,0 +1,44 @@
+package utils;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.base.Charsets;
+import com.google.gson.Gson;
+import feign.Response;
+import models.Post;
+import models.User;
+import org.apache.commons.io.IOUtils;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+
+
+public class ApiUtils {
+
+    public static String returnJsonString(Response response) throws IOException {
+        String jsonString = IOUtils.toString(response.body().asInputStream(), Charsets.UTF_8);
+        return jsonString;
+    }
+
+    public static void getAllPosts(Response response) throws IOException {
+        List<Post> allPosts = Arrays.asList(new ObjectMapper().readValue(returnJsonString(response), Post[].class));
+    }
+
+    public static String isListInJson(Response allPostsResponse){
+        Map<String, Collection<String>> headers = allPostsResponse.headers();
+        return headers.get("Content-Type").toString();
+    }
+
+    public static int returnStatus(Response emptyPostsResponse){
+        return emptyPostsResponse.status();
+    }
+
+    public static Post returnPost(Response response) throws IOException {
+        return new Gson().fromJson(returnJsonString(response), Post.class);
```

Comment:
you can leave only one method for POJO generating and use the generic format 
e.g. 
```
public static <T> T returnPost(String response, Class tClass) throws IOException {
        return (T) new Gson().fromJson(response, tClass.class);
        }
```

Reply:
it can be reused for different data